FaunaDB.Client/Query/Language.Miscellaneous.cs

```diff
@@ -439,5 +439,303 @@ public partial struct Language
+        /// </para>
+        /// </summary>
+        public static Expr Round(Expr expr, Expr precision) =>
+          UnescapedObject.With("round", expr, "precision", precision);
+
+        /// <summary>
+        /// Creates a new Sign expression.
+        /// <para>
+        /// See the <see href="https://docs.fauna.com/fauna/current/api/fql/functions/sign">FaunaDB Sign Function</see>
+        /// </para>
+        /// </summary>
+        public static Expr Sign(Expr expr) =>
+          UnescapedObject.With("sign", expr);
+
+        /// <summary>
+        /// Creates a new Sin expression.
+        /// <para>
+        /// See the <see href="https://docs.fauna.com/fauna/current/api/fql/functions/sin">FaunaDB Sin Function</see>
+        /// </para>
+        /// </summary>
+        public static Expr Sin(Expr expr) =>
+          UnescapedObject.With("sin", expr);
+
+        /// <summary>
+        /// Creates a new Sinh expression.
+        /// <para>
+        /// See the <see href="https://docs.fauna.com/fauna/current/api/fql/functions/sinh">FaunaDB Sinh Function</see>
+        /// </para>
+        /// </summary>
+        public static Expr Sinh(Expr expr) =>
+          UnescapedObject.With("sinh", expr);
+
+        /// <summary>
+        /// Creates a new Sqrt expression.
+        /// <para>
+        /// See the <see href="https://docs.fauna.com/fauna/current/api/fql/functions/sqrt">FaunaDB Sqrt Function</see>
+        /// </para>
+        /// </summary>
+        public static Expr Sqrt(Expr expr) =>
+          UnescapedObject.With("sqrt", expr);
+
+        /// <summary>
+        /// Creates a new Tan expression.
+        /// <para>
+        /// See the <see href="https://docs.fauna.com/fauna/current/api/fql/functions/tan">FaunaDB Tan Function</see>
+        /// </para>
+        /// </summary>
+        public static Expr Tan(Expr expr) =>
+          UnescapedObject.With("tan", expr);
+
+        /// <summary>
+        /// Creates a new Tanh expression.
+        /// <para>
+        /// See the <see href="https://docs.fauna.com/fauna/current/api/fql/functions/tanh">FaunaDB Tanh Function</see>
+        /// </para>
+        /// </summary>
+        public static Expr Tanh(Expr expr) =>
+          UnescapedObject.With("tanh", expr);
+
+        /// <summary>
+        /// Creates a new Trunc expression.
+        /// <para>
+        /// See the <see href="https://docs.fauna.com/fauna/current/api/fql/functions/trunc">FaunaDB Trunc Function</see>
+        /// </para>
+        /// </summary>
+        public static Expr Trunc(Expr expr) =>
+          UnescapedObject.With("trunc", expr);
+
+        /// <summary>
+        /// Creates a new Trunc expression.
+        /// <para>
+        /// See the <see href="https://docs.fauna.com/fauna/current/api/fql/functions/trunc">FaunaDB Trunc Function</see>
+        /// </para>
+        /// </summary>
+        public static Expr Trunc(Expr expr, Expr precision) =>
```

Comment:
you can remove the above overload and leave just this one with `Expr precision = null`